### ISSUE TITLE
Create HTML Basic Structure

### ISSUE THREAD

**Issue by MMortaga** [khaledWS/content-management-system#1]
- As a user, I should be able to navigate the website without facing any accessibility issues.
- As a user, I should have a navbar to easily navigate around the website.
